__mocks__/apollo-server-env.ts

```diff
@@ -15,8 +15,8 @@ import {
 } from '../packages/apollo-server-env';
 
 interface FetchMock extends jest.Mock<typeof fetch> {
-  mockResponseOnce(data?: any, headers?: HeadersInit, status?: number): void;
-  mockJSONResponseOnce(data?: object, headers?: HeadersInit): void;
+  mockResponseOnce(data?: any, headers?: HeadersInit, status?: number): this;
```

Comment:
This change enables setting up multiple mock invocations. I believe `void` is technically an incorrect return type as the underlying API used here actually returns `this`. See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jest/index.d.ts#L997.

Reply:
👍